azure-pipelines.yml

```diff
@@ -190,6 +190,20 @@ jobs:
       modifyEnvironment: true
     condition: and(succeeded(), eq(variables['vmImage'], 'windows-2019'))
 
+  - powershell: |
```

Comment:
We want opencv to be an optional dependency. Also, on Linux, it's pretty easy to get the libraries without the source, which is nice because OpenCV is massive, so we don't really want it as a submodule for Linux.